app/controllers/static_pages_controller.rb

```diff
@@ -28,7 +28,11 @@ def privacy
 
 
   def show
-    web = current_user.websites
+    if params[:collection_id].present?
+      web = current_user.collections.find_by_id(params[:collection_id]).try(:websites)
+    else
+      web = current_user.websites
+    end
```

Comment:
- Let's call the variable `websites`.
- Is the query in line 32 the best way to query it? Can we do it more clearly like this:

```ruby
def show
  websites = current_user.websites
  websites = websites.where(collection_id: params[:collection_id]) if params.fetch(:collection_id, nil)
  # ... rest of the method
end
```